src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -56,7 +56,7 @@ public EditCommand parse(String args) throws ParseException {
             editPersonDescriptor.setEmail(ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get()));
         }
         if (argMultimap.getValue(PREFIX_ADDRESS).isPresent()) {
-            editPersonDescriptor.setAddress(ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get()));
+            editPersonDescriptor.setAddress(ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).orElse("")));
```

Comment:
I could be misunderstanding this but if `argMultimap.getValue(PREFIX_ADDRESS).isPresent()` is `true`, then `editPersonDescriptor.setAddress(ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).orElse("")))` will always return the `argMultimap.getValue(PREFIX_ADDRESS)` value, even if it is a blank string (ie: redundency...?). 

Please correct me if I am wrong